bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WZTo2L2Q01j_5f_NLO_FXFX/WZTo2L2Q01j_5f_NLO_FXFX_run_card.dat

```diff
@@ -93,6 +97,8 @@ $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
   0.5     = rw_Fscale_down   ! lower bound for fact scale variations
   2.0     = rw_Fscale_up     ! upper bound for fact scale variations
 $DEFAULT_PDF_MEMBERS = reweight_PDF     ! reweight to get PDF uncertainty
+      ! First of the error PDF sets
+      ! Last of the error PDF sets
```

Comment:
remove from reweight_PDF to here.
You double defined reweight_PDF 

and below 2 shouldn't exist (although it's commented  out)
       ! First of the error PDF sets
       ! Last of the error PDF sets